src/index.html

```diff
@@ -1,12 +1,374 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
```

Comment:
I did this in order to set padding and margins for the header, bady and section. because according to it I can’t simply set the outer margins for the element